package/doc/sphinx/source/documentation_pages/trajectory_transformations.rst

```diff
@@ -0,0 +1,69 @@
+.. Contains the formatted docstrings for the transformations located in 'mdanalysis/MDAnalysis/transformations'
+.. _transformations:
+
+**************************
+Trajectory transformations
+**************************
+
+.. module:: MDAnalysis.transformations
+
+The transformations submodule contains a collection of functions to modify the 
+trajectory. Coordinate transformations, such as PBC corrections and molecule fitting
+are often required for some analyses and visualization, and the functions in this 
+module allow transformations to be applied on-the-fly. 
+These transformation functions can be called by the user for any given
+timestep of the trajectory, added as a workflow using :meth:`add_transformations`
+of the :mod:`~MDAnalysis.coordinates.base` module, or upon Universe creation using
+the keyword argument `transformations`. Note that in the two latter cases, the 
+workflow cannot be changed after being defined.
+
+In addition to the specific arguments that each transformation can take, they also 
+contain a wrapped function that takes a `Timestep` object as argument. 
+So, a transformation can be roughly defined as follows:
+
+.. code-block:: python
+
+    def transformations(*args,**kwargs):
+        # do some things
+            def wrapped(ts):
+                # apply changes to the Timestep object
+                return ts
+            
+            return wrapped
+
+
+See :func:`MDAnalysis.transformations.translate` for a simple example.    
+    
+
+.. rubric:: Examples
+
+e.g. translate the coordinates of a frame:
```

Comment:
*e.g.* in italic. However, the *e.g,* could go completely and the line could be

    Translating the coordinates of a frame: